cli/util/osfs.go

```diff
@@ -0,0 +1,25 @@
+package util
+
+import (
+	"io/fs"
+	"os"
+)
+
+type osFS struct{}
+
+// Default implementation of fs.FS interface. Interface fs.FS should be added as an argument
+// to any function where you need to be able to substitute non-default FS. The most obvious
+// scenario is using mock FS for testing. In such a case general code might use this default
+// implementation, while the tests will use some mock FS implementation (like fstest.MapFS).
+
+func GetOsFS() fs.FS {
```

Comment:
It would be in godoc style:
```suggestion
// GetOsFS returns a default implementation of fs.FS interface. Interface fs.FS should be
// added as an argument to any function where you need to be able to substitute non-default
// FS. The most obvious scenario is using mock FS for testing. In such a case general code
// might use this default implementation, while the tests will use some mock FS
// implementation (like fstest.MapFS).
func GetOsFS() fs.FS {
```
The beginning is very important here + there is should not be an empty line between comment and a function.
```
GetOsFS returns
```